docs/user/debugging.md

```diff
@@ -71,28 +79,132 @@ More information can be found in the [View Logs with Logcat][] help article.
 
 [View Logs with Logcat]: https://developer.android.com/studio/debug/am-logcat
 
-### Command line
+#### Command line
 
 On the command line you can show all of the log output using:
 
-```
+```shell
 adb logcat
 ```
 
 This is the unfiltered output of all log messages.
 You can match for `glean` using grep:
 
-```
+```shell
 adb logcat | grep -i glean
 ```
 
 A simple way to filter for only the application that is being debugged is by using [pidcat][], a wrapper around `adb`, which adds colors and proper filtering by application ID and log level.
 Run it like this to filter for an application:
 
-```
+```shell
 pidcat [applicationId]
 ```
 
 In the above `[applicationId]` is the product's application id as defined in the manifest file and/or build script. For the Glean sample application, this is `org.mozilla.samples.glean` for a release build and `org.mozilla.samples.glean.debug` for a debug build.
 
 [pidcat]: https://github.com/JakeWharton/pidcat
+
+## Debugging the Glean SDK in iOS
+
+Unlike Android, we aren't able to export a View or similar to the `GleanDebugActivity` in Android that can process debugging commands.  Instead, iOS allows for implementation of a custom URL scheme _within the application_.  Instead of launching the application through a proxy like `GleanDebugActivity`, we can launch the application from a URL with the debugging commands embedded in the URL itself.
```

Comment:
```suggestion
iOS allows for implementation of a custom URL scheme _within the application_, so that users can launch the application from a URL with the debugging commands embedded in the URL itself.
```

Reply:
I'd personally like to keep the comparison between platforms out: in the log run, we might want a script for enabling consistent testing flows across platforms. See [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1568656)

Reply:
Not a problem, my first pass at this was more conversational and I had to go back and refactor it, this is an artifact of that.  I can remove the comparisons to Android functionality.